packages/langium/src/lsp/language-server.ts

```diff
@@ -610,7 +611,7 @@ export function createRequestHandler<P extends { textDocument: TextDocumentIdent
             console.error(`Could not find service instance for uri: '${uri.toString()}'`);
             return null;
         }
-        const document = documents.getOrCreateDocument(uri);
+        const document = documents.getDocument(uri);
```

Comment:
This function is `async`, so we could `await documents.getOrCreateDocument(uri)`?